python/bindings/openravepy_int.cpp

```diff
@@ -2764,16 +2764,16 @@ py::object PyEnvironmentBase::GetDescription() const
     return ConvertStringToUnicode(_penv->GetDescription());
 }
 
-void PyEnvironmentBase::SetKeywords(const std::vector<std::string>& sceneKeywords)
+void PyEnvironmentBase::SetKeywords(py::object oSceneKeywords)
```

Comment:
@yoshikikanemoto not sure, was this one required? only GetKeywords fix was required for me.

(note: I can test only pybind11 now)

Reply:
yes, it was required